### PR TITLE
Upgrade/coverage

### PR DESCRIPTION
**Pull request by mmumby**

### CODE REVIEW

File: frontend/src/tests/selectors/mineSelectors.spec.js [hunk 1]
```diff
@@ -1,29 +1,118 @@
...
+    mineGuid = Mock.MINES.mineIds[1]
+    const selected = getCurrentPermittees.resultFunc(mines, mineGuid);
+    expect(selected).toEqual((Mock.PERMITTEE.permittees));
+  });
 
-    expect(getMineIds(mockState)).toEqual(["test123", "test456"]);
+  it('`getCurrentPermitteeIds` calls `mineReducer.getCurrentPermitteeIds`', () => {
+    mines =  Mock.MINES.mines;
+    mineGuid = Mock.MINES.mineIds[1]
+    const selected = getCurrentPermitteeIds.resultFunc(mines, mineGuid)
```
Comment (NWCalvank) [inline frontend/src/tests/selectors/mineSelectors.spec.js]: Semicolons.

Okay. I thiiiink I see how these tests are working.. They essentially just test that `getCurrentPermitteeIds` returns the expected value, which infers that `.getCurrentPermitteeIds` was called in the process?

File: frontend/src/tests/selectors/modalSelectors.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import {getIsModalOpen, getProps, getContent} from '@/selectors/modalSelectors';
+import modalReducer from '@/reducers/modalReducer';
+import {openModal, closeModal} from '@/actions/modalActions';
+import {MODAL} from '@/constants/reducerTypes';
+import * as Mock from '@/tests/mocks/dataMocks';
+import { modalConfig } from '@/components/modalContent/config';
```
Comment (NWCalvank) [inline frontend/src/tests/selectors/modalSelectors.spec.js]: Inconsistent import destructuring spacing 😢

File: frontend/src/tests/selectors/modalSelectors.spec.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+const mockModal = {
+  isModalOpen: true,
+  props: [Mock.MINES.mines[Mock.MINES.mineIds[0]],  Mock.REGION_OPTIONS, Mock.STATUS_OPTIONS],
+  content: modalConfig.MINE_RECORD,
+}
```
Comment (NWCalvank) [inline frontend/src/tests/selectors/modalSelectors.spec.js]: Semicolon

File: frontend/src/tests/mocks/dataMocks.js [hunk 4]
```diff
@@ -7,6 +7,147 @@ export const createMockHeader = () => ({
 
 export const ERROR = { error: {message: "Errors", status: 400}}
 
+// used for testing selectors
+export const MINE_RESPONSE = {
```
[the remark this answers is not in the document]
Comment (FreshGyan) [reply frontend/src/tests/mocks/dataMocks.js]: That sounds like an excellent idea! If you would like to create a ticket and inform AAron about it, we can pull it into a sprint :D

**Pull request merged**